src/devrules/tui/services/github_service.py

```
@@ -103,3 +120,180 @@ def is_configured(self) -> bool:
+                comment = GitHubComment(
+                    id=item["id"],
+                    body=item["body"],
+                    author=comment_user,
+                    created_at=item["created_at"],
+                    issue_number=issue_data["number"],
+                    issue_title=issue_data["title"],
+                    html_url=item["html_url"],
+                )
+                recent_comments.append(comment)
+            except Exception:
+                continue
```

Ruff (0.14.14)

`try`-`except`-`continue` detected, consider logging the exception

(S112)